### PR TITLE
Merge feature S02 S03

### PR DESCRIPTION
**Pull request by Bravax**

### CODE REVIEW

File: frontend/src/components/AboutUs/AboutUsPage.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      name: "Romain IMBRASSE",
+      imgSrc: Romain,
+      github: "https://github.com/Romlion",
+      linkedin: "https://www.linkedin.com/in/romain-imbrasse-9480ab1ba/",
+    },
+  ];
+  return (
+    <>
+      <Link to="/">
+        <button type="button" className="closeButtonAboutUs">
```
Comment (Yoann-Fortin) [inline frontend/src/components/AboutUs/AboutUsPage.jsx]: The button is not necessary, instead, an aria-label is required.

File: frontend/src/components/AboutUs/AboutUsPage.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.closeButtonAboutUs {
+  position: absolute;
+  left: 0;
+  margin-top: 2vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/AboutUs/AboutUsPage.scss]: Not necessary with absolute position

File: frontend/src/components/AboutUs/AboutUsPage.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  border-radius: 200px;
+  border: 3px solid $primaryColor;
+  background-color: $secondaryColor;
+  height: 3.5vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/AboutUs/AboutUsPage.scss]: Not vw to a height

File: frontend/src/components/AboutUs/AboutUsPage.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  width: 4vw;
+  align-items: center;
+  color: $primaryColor;
+  font-size: 2vw;
+  margin-left: 2vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/AboutUs/AboutUsPage.scss]: Not necessary with absolute position

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+    }
+  }
+}
+.playDevice {
+  all: unset;
+}
+.closeButtonChooseLevelPage {
+  position: absolute;
+  right: 0;
+  margin-top: 20px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: Not with absolute position

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+  border-radius: 200px;
+  border: 3px solid $primaryColor;
+  background-color: $secondaryColor;
+  height: 1rem;
+  width: 1rem;
+  font-size: 0.7rem;
+  align-items: center;
+  color: $primaryColor;
+  font-size: 0.8rem;
+  margin-right: 2rem;
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: Not with absolute position

File: frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.jsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+import Story from "./Story/Story";
+import "./ContainerStoryRules.scss";
+
+function ContainerStoryRules() {
+  return (
+    <section className="storyRules">
+      <Story />
+      <Rules />
+      <Link to="/ChooseLevelPage">
+        <button type="button" className="storyRules__playButton">
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.jsx]: Button isn't necessary

File: frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.scss
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/Rules/Rules.scss:1]: To delete

File: frontend/src/components/HomePage/ContainerStoryRules/Story/Story.scss
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/ContainerStoryRules/Story/Story.scss:1]: To delete

File: frontend/src/components/HomePage/MainHeader/MainHeader.jsx [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import "./MainHeader.scss";
+import { Link } from "react-router-dom";
+
+function MainHeader() {
+  return (
+    <header>
+      <h1 className="mainTitle">Le collectionneur temporel</h1>
+      <Link to="/aboutus">
+        <button type="button" className="aboutButton">
```
Comment (Yoann-Fortin) [inline frontend/src/components/HomePage/MainHeader/MainHeader.jsx]: Button isn't necessary

File: frontend/src/components/LevelPage/GameMenu/GameMenu.scss [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+hr {
+  border: solid 2px black;
+  width: 100%;
+}
+
+nav {
+  display: flex;
+  flex-direction: column;
+  width: 20vw;
+  border-right: solid 5px #2b160f;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/GameMenu.scss]: A color variable is better

File: frontend/src/components/LevelPage/GameMenu/GameMenu.scss [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+.hidden {
+  display: none;
+}
+
+.closeNav {
+  position: absolute;
+  border: 2.5px solid $primaryColor;
+  background-color: $secondaryColor;
+  border-radius: 200px;
+  color: white;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/GameMenu.scss]: a color variable is better

File: frontend/src/components/LevelPage/GameMenu/GameMenu.scss [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+  width: 3vw;
+  font-size: 1.5vw;
+  height: 5.5vh;
+  margin-top: 0.5vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/GameMenu.scss]: Not necessary with absolute position

File: frontend/src/components/LevelPage/GameMenu/GameMenu.scss [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+  left: 15%;
+  top: 50%;
+}
+
+@media screen and (min-width: 940px) {
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/GameMenu.scss]: to inverse with antother media query

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+  position: absolute;
+  left: 30%;
+  top: 25%;
+  background-color: $secondaryColor;
+  border: 5px solid $optionalColor;
+  color: $primaryColor;
+  width: 50%;
+  height: 50%;
+  text-align: center;
+  margin: 0 auto;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss]: Why margin with an absolute position here ?

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+  transition: opacity 0.3s;
+
+  &__closeButton {
+    position: relative;
+    left: 50%;
+    transform: translateX(-200%);
+  }
+
+  &__show {
+    border: 10px solid #8b4513;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss]: A color variable is better

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+    border-radius: 15px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
+    background-color: #d2b48c;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss]: a color variable is better

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+  top: 0;
+  left: 0;
+}
+
+.bag {
+  width: 10vw;
+}
+
+.invButton {
+  border: 0;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.scss]: Dead code

File: frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.scss [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+  position: absolute;
+  left: 30%;
+  top: 25%;
+  background-color: $secondaryColor;
+  border: 5px solid $optionalColor;
+  color: $primaryColor;
+  width: 50%;
+  height: 50%;
+  text-align: center;
+  margin: 0 auto;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.scss]: dead code

File: frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.scss [hunk 18]
```diff
@@ -0,0 +1,58 @@
...
+  top: 0;
+  left: 0;
+}
+
+.noteBook {
+  width: 10vw;
+}
+
+.objButton {
+  border: 0;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.scss]: dead code

File: frontend/src/components/LevelPage/LevelPage.scss [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.closeButton {
+  position: relative;
+}
+
+.closeButtonLevelPage {
+  position: absolute;
+  right: 0;
+  margin-top: 20px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/LevelPage.scss]: Not necessary with an absolute position

File: frontend/src/components/LevelPage/LevelPage.scss [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+  border-radius: 200px;
+  border: 3px solid $primaryColor;
+  background-color: $secondaryColor;
+  height: 2rem;
+  width: 2rem;
+  align-items: center;
+  color: $primaryColor;
+  font-size: 1rem;
+  margin-right: 2rem;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/LevelPage.scss]: Not necessary with absolute position

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.jsx [hunk 21]
```diff
@@ -0,0 +1,45 @@
...
+
+function ReturnGameMenu() {
+  const [popupMessage, setPopupMessage] = useState("");
+  function HandleEvent() {
+    setPopupMessage(
+      <article className="messagePopUp">
+        Voulez-vous vraiment revenir au Menu Principal ?
+        <div className="areYouSureButton">
+          <Link to="/">
+            <button type="button" className="backtoMainMenu">
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.jsx]: button isn't necessary

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.jsx [hunk 22]
```diff
@@ -0,0 +1,45 @@
...
+import { useState } from "react";
+import "./ReturnGameMenu.scss";
+
+function ReturnGameMenu() {
+  const [popupMessage, setPopupMessage] = useState("");
+  function HandleEvent() {
+    setPopupMessage(
+      <article className="messagePopUp">
+        Voulez-vous vraiment revenir au Menu Principal ?
+        <div className="areYouSureButton">
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.jsx]: a div ? why ?

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss [hunk 23]
```diff
@@ -0,0 +1,61 @@
+@import "../../../variables.scss";
+
+.backMainMenuButton {
+  position: absolute;
+  right: 0;
+  margin-top: 2vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss]: Not necessary with an absolute position

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss [hunk 24]
```diff
@@ -0,0 +1,61 @@
...
+  border-radius: 200px;
+  border: 0.2vw solid $primaryColor;
+  background-color: $secondaryColor;
+  height: 3.5vh;
+  width: 2vw;
+  align-items: center;
+  color: $primaryColor;
+  font-size: 0.8vw;
+  margin-right: 2vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss]: Not necessary with an absolute position

File: frontend/src/components/LevelPage/TargetButton/TargetButton.scss [hunk 25]
```diff
@@ -0,0 +1,81 @@
...
+.ptero {
+  width: 10vw;
+  height: 10vh;
+  z-index: 1;
+}
+
+.pteroEgg {
+  width: 2vw;
+  height: 2vh;
+  z-index: 0;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButton.scss]: dead code

File: frontend/src/components/LevelPage/TargetButton/TargetButton.scss [hunk 26]
```diff
@@ -0,0 +1,81 @@
...
+  font-size: 3vw;
+}
+
+.info {
+  font-size: 0.5vw;
+  flex: 1;
+  border-left: 2px solid $secondaryColor;
+  border-right: 2px solid $secondaryColor;
+  height: 1.5vh;
+  margin-right: 20px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButton.scss]: responsive value is better

File: frontend/src/variables.scss [hunk 27]
```diff
@@ -0,0 +1,9 @@
+@import url("https://fonts.googleapis.com/css2?family=Iceberg&display=swap");
+
+$fontFamily: "Iceberg";
```
Comment (Yoann-Fortin) [inline frontend/src/variables.scss:3]: add a sans-serif font

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Cyrildab — APPROVED** [review]

**Comment by Bravax** [comment]
Resolved problems

**Review by Yoann-Fortin — APPROVED** [review]

**Review by ItsMatDev — APPROVED** [review]

**Review by pagouillart — APPROVED** [review]

**Pull request merged**